gradle-plugin/src/main/groovy/grails/views/gradle/util/GrailsNameUtils.groovy

```diff
@@ -0,0 +1,59 @@
+package grails.views.gradle.util
+
+import groovy.transform.CompileStatic
+
+/**
+ * This class is copying some methods from GrailsNameUtils in grails-bootstrap.
+ * This is done to avoid a dependency on grails-bootstrap.
+ * GrailsNameUtils in grails-bootstrap is probably a candidate for moving to a future "grails-common" module.
+ */
+@CompileStatic
+class GrailsNameUtils {
```

Comment:
There is a ticket to extract common files into a common project.  Can you link to this file in that ticket?

Reply:
https://github.com/grails/grails-core/issues/13850#issuecomment-2558122348